components/bl-simpleModal-component/src/index.js

```diff
@@ -0,0 +1,94 @@
+import { useState, useEffect } from 'react';
+import { SimpleModalButton, SimpleModalContainer } from './subcomponents';
+
+export default function MyCustomComponent({ component, eventHandlers }) {
+  const {
+    display,
+    classList,
+    titleModal,
+    textModal,
+    typeSimpleModal,
+    placeholderText,
+    closeButtonLabel,
+    submitButtonLabel,
+    closingDuration,
+  } = component;
+  const { onCloseButton, onSubmitButton, onInputValueChange } = eventHandlers;
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isClosing, setIsClosing] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const simpleModalClasses = useSimpleModalClasses(isClosing);
+
+  component.closeModal = () => {
+    setIsClosing(false);
```

Comment:
When we close the modal window that must be in 'closing' state, so for me, it's not logical when we close the modal window but set isClosing to false.